presto-tests/src/test/java/io/prestosql/execution/TestEventListener.java

```diff
@@ -58,6 +58,7 @@
         extends AbstractTestQueryFramework
 {
     private static final int SPLITS_PER_NODE = 3;
+    private static final String IGNORE_EVENT_MARKER = "-- ignore_generated_event";
```

Comment:
should not be prt of "Add query filtering logic to EventsBuilder" commit

Reply:
I put it in test for now. Only one tests uses it. 
In `EventBuilder` you can provide any predicate which can assess metadata, not necessarily look for substring in query.